packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/model/Model.java

```diff
@@ -69,4 +76,19 @@ public Model(Model other) {
         this.semantics = other.semantics;
         this.metadata = other.metadata;
     }
+
+	@TSIgnore
+	public List<ModelParameter> getParameters() {
+		ObjectMapper objectMapper = new ObjectMapper();
+		if(this.isRegnet()) {
+			return objectMapper.convertValue(this.getModel().get("parameters"), new TypeReference<List<ModelParameter>>() {});
+		} else {
+			return this.getSemantics().getOde().getParameters();
+		}
+	}
+
+	@TSIgnore
+	public boolean isRegnet() {
+		return this.getHeader().getSchemaName().toLowerCase().equals("regnet");
```

Comment:
Don't worry about changing it because this is fine, but just as an FYI Java String has a `equalsignorecase`

Reply:
good to know!